--- origsrc/clucene-core-0.9.21b/src/CLucene/config/CompilerGcc.h	2008-10-23 12:44:36.000000000 -0500
+++ src/clucene-core-0.9.21b/src/CLucene/config/CompilerGcc.h	2009-04-29 20:40:00.889000000 -0500
@@ -65,10 +65,7 @@
     typedef unsigned short wchar_t;
 #endif
 
-#if defined(__CYGWIN__)
-   //cygwin seems to incorrectly define that it has wprintf???
-   #undef _CL_HAVE_WPRINTF
-#elif defined(__MINGW32__)
+#if defined(__MINGW32__)
 	#ifndef _CL_HAVE_WINDOWS_H
 		#define _CL_HAVE_WINDOWS_H
 	#endif
